js/global.js

```diff
@@ -1878,7 +1877,7 @@ jQuery(document).ready(function($) {
     // calls this function directly when in wysiwyg format, as it needs to
     // handle an iframe, and the editor instance needs to be referenced.
     if ($.fn.atwho && gdn.atCompleteInit) {
-        $(document).on('EditCommentFormLoaded popupReveal', function() {
+        $(document).on('contentLoad', function() {
             gdn.atCompleteInit('.BodyBox', '');
```

Comment:
All contentLoad handlers need to declare the e parameter and operate on the e.target argument. Make sure you step through the code and see that you aren't applying stuff to the document over and over again.